Griddly.NetCore.Razor/Pages/Shared/Griddly/Griddly.cshtml

```diff
@@ -391,9 +391,9 @@
     @if (settings.FooterTemplate == null || simple)
     {
         <span class="griddly-summary">
-            <span class="@(settings.Css.IsBootstrap4 ? "d-none d-sm-inline" : "hidden-xs")">Records</span>
+            <span class="@(settings.Css.Is(CssFramework.Bootstrap4, CssFramework.Bootstrap5) ? "d-none d-sm-inline" : "hidden-xs")">Records</span>
```

Comment:
What about Css.Ge(CssFramework.Bootstrap4) ? Since this seems to be a commonly used query.

Reply:
I thought about doing something like that, but I also didn't want to box us into a "bootstrap only" corner. By being explicit we could support other frameworks in the future.